src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,61 @@
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  column-gap: 20px;
+
+  @include onTablet {
```

Comment:
It is a good practice to provide comments explaining the purpose of the breakpoints (@include onTablet and @include onDesktop) for better understanding. Example: // Tablet breakpoint